pom.xml

```diff
@@ -377,21 +372,6 @@
 			<version>2.1.1</version>
 		</dependency>
 		<dependency>
-			<groupId>org.seamless</groupId>
-			<artifactId>seamless-http</artifactId>
-			<version>1.1.1</version>
-		</dependency>
-		<dependency>
-			<groupId>org.seamless</groupId>
```

Comment:
@valib Many of the classes used by Cling is from Seamless, it's by the same author and Seamless is the his "common code" used both in Cling and other projects. Debugging Cling will also get harder without access to this source code, so I think they should stay. To save space in the compiled binary, using the shade plugin would be a better solution.